src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -363,4 +363,16 @@
   <data name="PX1080Title" xml:space="preserve">
     <value>Data view delegates should not start long-running operations</value>
   </data>
+  <data name="PX1055Fix1" xml:space="preserve">
+    <value>Make the field bound to identity column the only key field in the DAC</value>
+  </data>
+  <data name="PX1055Fix2" xml:space="preserve">
+    <value>Alter the definition of the field bound to identity column to make it a non-key field and leave all other key fields untouched</value>
+  </data>
+  <data name="PX1055Fix3" xml:space="preserve">
+    <value>Remove the definition of the field bound to identity column and leave all other key fields untouched (it will be also very helpful to show a hint suggesting to drop the identity column, to which this key field is bound, from database)</value>
+  </data>
+  <data name="PX1055Title" xml:space="preserve">
+    <value>Invalid primary key of the DAC. Use a single identity field or one or multiple natural fields for the key.</value>
+  </data>
```

Comment:
These strings are needed to be reviewed, especially PX1055Fix3)

Reply:
@saloponov , I reviewed these strings yesterday but I cannot see my comments anywhere. Did you receive them?

Reply:
@saloponov , the comments were in this pull request: https://github.com/Acumatica/Acuminator/pull/120

Reply:
@k-s-popova , thanks! I didn't see comments in #120(